tests/e2e/e2e_ignore_namespace_test.go

```diff
@@ -0,0 +1,197 @@
+package e2e
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openservicemesh/osm/pkg/constants"
+	. "github.com/openservicemesh/osm/tests/framework"
+)
+
+var _ = OSMDescribe("Ignore Namespaces",
+	OSMDescribeInfo{
+		Tier:   1,
+		Bucket: 2,
```

Comment:
Will update to `Tier: 2` after PR review is complete

Reply:
The Tier doesn't matter anymore. All tests run for PR workflows.